core/src/apps/bitcoin/sign_tx/layout.py

```diff
@@ -183,13 +183,23 @@ async def confirm_total(
     ctx: wire.Context,
     spending: int,
     fee: int,
+    fee_rate: float,
     coin: CoinInfo,
     amount_unit: AmountUnit,
 ) -> None:
+    fee_rate_str = ""
```

Comment:
maybe `fee_rate_str: str | None = None` would be more appropriate?

Reply:
828de33ac491da08382cd8954719419807bfb153